docs/operations/images.md

```diff
@@ -164,6 +178,19 @@ aws ec2 describe-images --region us-east-1 --output table \
   --filters "Name=name,Values=ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-*"
 ```
 
+### Ubuntu 22.04 (Jammy)
+
+Ubuntu 22.04 is based on Kernel version **5.15** which fixes all the known major Kernel bugs.
```

Comment:
I feel this statement may age poorly :-)

Reply:
😂